sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -4,29 +4,22 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// Configurations that allow callers to specify details about how to execute
-    /// an Extractive Text Summarization action in a set of documents.
-    /// For example, set the model version, specify the order in which extracted
-    /// sentences are returned, and more.
+    /// A set of options used to configure extractive summarization, including the model version to use, the maximum
+    /// number of sentences to extract, and more.
     /// </summary>
     public class ExtractSummaryAction
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
-        /// class which allows callers to specify details about how to execute
-        /// an extractive text summarization action in a set of documents.
-        /// For example, set the model version, specify the order in which extracted
-        /// sentences are returned, and more.
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/> class.
         /// </summary>
         public ExtractSummaryAction()
         {
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
-        /// class based on the values of a <see cref="ExtractSummaryOptions"/>.
-        /// It sets the <see cref="ModelVersion"/>, <see cref="DisableServiceLogs"/>,
-        /// <see cref="MaxSentenceCount"/>, and <see cref="OrderBy"/> properties.
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/> class based on the given
+        /// <see cref="ExtractSummaryOptions"/>. It sets the <see cref="ModelVersion"/>,
+        /// <see cref="DisableServiceLogs"/>, <see cref="MaxSentenceCount"/>, and <see cref="OrderBy"/> properties.
```

Comment:
IMO, no need to talk about the implementation that may change and the comment falls out of date. Generally, "Initialize a new instance..." doesn't go into details, not even for the .NET BCL itself.